src/types/redis_bloom_chain.cc

```diff
@@ -95,26 +111,12 @@ void BloomChain::bloomAdd(const Slice &item, std::string *bf_data) {
   *bf_data = std::move(block_split_bloom_filter).GetData();
 }
 
-rocksdb::Status BloomChain::bloomCheck(const Slice &bf_key, const std::vector<Slice> &items,
-                                       std::vector<bool> *exists) {
-  LatestSnapShot ss(storage_);
-  rocksdb::ReadOptions read_options;
-  read_options.snapshot = ss.GetSnapShot();
-  std::string bf_data;
-  rocksdb::Status s = storage_->Get(read_options, bf_key, &bf_data);
-  if (!s.ok()) return s;
+bool BloomChain::bloomCheck(const Slice &item, std::string &bf_data) {
   BlockSplitBloomFilter block_split_bloom_filter;
-  block_split_bloom_filter.Init(std::move(bf_data));
+  block_split_bloom_filter.Init(bf_data);
```

Comment:
Maybe you can refactor `BlockSplitBloomFilter` and give it a non-owned slice version. Currently, `bloomCheck` will copy the whole bf_data, which could be cpu-consuming and waste